text/0095-nft-standard.md

```diff
@@ -0,0 +1,371 @@
+# Casper NFT Standard Proposal
+
+Casper NFT Standard Proposal (Inspired by ERC-721 & adapted for the Casper ecosystem)
+
+## Summary
+
+CEP PR: [casperlabs/ceps#0095](https://github.com/casper-network/ceps/pull/95)
+
+This standard defines a unified API for implementing non-fungible tokens (NFTs) on the Casper network. It introduces core functionality for tracking and transferring NFTs in a consistent and interoperable way.
+
+We have taken into account use cases where NFTs are owned directly by individuals, as well as scenarios where ownership and transfers are managed by third-party entities such as marketplaces, wallets, or custodial services ("operators"). NFTs on Casper can represent ownership of both digital and physical assets, and the range of potential applications continues to grow:
+
+- Physical assets — real estate, unique artwork 
+- Digital collectibles — rare digital items, game assets, virtual companions 
+- Obligations and non-traditional assets — debt instruments, legal contracts, responsibilities
+
+Each NFT is uniquely identifiable, and as such, must be tracked individually. Unlike fungible tokens, NFTs are not interchangeable — every item is distinct, and this standard ensures that uniqueness and ownership are verifiable and transferable on-chain.
+
+## Motivation
+
+[motivation]: #motivation
+A standard interface for non-fungible tokens (NFTs) on the Casper network enables wallets, marketplaces, decentralized applications, and other ecosystem tools to interact seamlessly with any compliant NFT contract. The lessons learned from Ethereum’s ERC-721 standard show that a consistent interface increases interoperability and speeds up adoption. On Casper, this standard will ensure that every NFT, whether tracking digital collectibles or physical assets, is uniquely identifiable and transferable.
+
+## Objective
+
+[objective]: #objective
+
+The objective of this proposal is to define a Casper-compatible NFT standard that supports:
+
+- Tracking ownership and metadata of NFTs. 
+- Transferring NFTs safely between accounts. 
+- Managing per-token approvals as well as operator-level approvals.
+- (Optionally) exposing metadata (name, symbol, token URI) so that applications can easily display NFT details.
```

Comment:
I'd remove or change this line. NFT metadata is about describing the underlying asset, rather than having a possibility to display it. 

The display problem is handled by providing a recommended convention for standard NFT metadata properties which can be found below in the standard.